index.html

```diff
@@ -2,23 +2,42 @@
 <html>
   <head>
     <title>Coding Basics</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Merriweather:wght@300&display=swap"
+      rel="stylesheet"
+    />
     <style>
       * {
         box-sizing: border-box;
       }
 
       body {
-        font-family: sans-serif;
+        font-family: "Merriweather", serif;
         margin-left: 30px;
         margin-right: 30px;
       }
 
+      table {
+        width: 100%;
+        margin-top: 1em;
+        margin-bottom: 1em;
+        border-collapse: collapse;
+        border: 1px;
+        text-align: center;
+      }
+
+      td {
+        padding: 0.2em;
+        border: 1px solid black;
+      }
+
       #header {
         text-align: center;
       }
 
       #container {
-        background-color: pink;
+        background-color: #b3cdd1;
+        background-image: linear-gradient(315deg, #b3cdd1 0%, #9fa4c4 74%);
```

Comment:
Beautiful gradient! Glad to see you stepping out of your comfort zone to try this :)